src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,233 @@
+    "fields":[{"name":"refCnt", "allowUnsafeAccess":true}]
+  },
+  {
+    "name": "io.lettuce.core.AbstractRedisAsyncCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.ChannelGroupListener",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.ConnectionEventTrigger",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.PlainChannelInitializer$1",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.RedisAsyncCommandsImpl",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.RedisClient",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.api.sync.BaseRedisCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.api.sync.RedisCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.api.sync.RedisGeoCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisHLLCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.api.sync.RedisHashCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.api.sync.RedisKeyCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name": "io.lettuce.core.api.sync.RedisListCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisScriptingCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisServerCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisSetCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisSortedSetCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisStreamCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisStringCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"io.lettuce.core.api.sync.RedisTransactionalCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+
+  {
+    "name":"io.lettuce.core.cluster.api.sync.RedisClusterCommands",
+    "allPublicMethods": true,
+    "allDeclaredConstructors":true
+  },
+
+  {
+    "name":"io.lettuce.core.protocol.CommandHandler",
```

Comment:
We also need:

* PubSubCommandHandler
* SslChannelInitializer
* SslRedisHandshakeHandler (6.0)
* RedisHandshakeHandler (6.0)

Not sure about `CommandEncoder`